env/fs_posix.cc

```diff
@@ -909,7 +906,8 @@ class PosixFileSystem : public FileSystem {
       return IOError("While doing statvfs", fname, errno);
     }
 
-    *free_space = ((uint64_t)sbuf.f_bsize * sbuf.f_bfree);
+    *free_space =
+        ((uint64_t)sbuf.f_bsize * (geteuid() ? sbuf.f_bavail : sbuf.f_bfree));
```

Comment:
Can you break this up into an if/else and add a comment as to what you are doing in the different cases?

Reply:
@mrambacher Done.  Please let me know if more edits desired.